packages/lesswrong/lib/karmaChanges.js

```diff
@@ -97,16 +147,19 @@ export function getKarmaChangeDateRange({settings, now, lastOpened=null, lastBat
       // Check whether the last time you opened the menu was in the same batch-period
       const openedBeforeNextBatch = lastOpened && lastOpened > lastDailyReset.toDate()
 
-      // If you open the notification menu again before the next batch has started, just return 
-      // the previous batch 
+      // If you open the notification menu again before the next batch has started, just return
+      // the previous batch
       if (previousBatchExists && openedBeforeNextBatch) {
         // Since we know that we reopened the notifications before the next batch, the last batch
         // will have ended at the last daily reset time
-        const lastBatchEnd = lastDailyReset 
-        return {
-          start: lastBatchStart,
-          end: lastBatchEnd.toDate()
-        };
+        const lastBatchEnd = lastDailyReset
```

Comment:
Something in all of this breaks the thing where you want to always reopen the previous batch. Not sure what.